### ISSUE TITLE
Great Theme! A few suggestions

### ISSUE THREAD

**Issue by chibicode** [wangweixuan/yithemes#1]
👍 Just wanted to let you know I've added this theme to this article:

https://medium.com/@chibicode/the-best-visual-studio-code-dark-and-light-themes-july-2018-edition-a9c2cc9548da

**Comment by chibicode** [comment]
Just curious: I think black text on dark purple background is hard to read. What do you think?

#### Suggestions

<img width="442" alt="screen shot 2019-02-12 at 11 52 34 am" src="https://user-images.githubusercontent.com/992008/52663955-b9f3c200-2ebc-11e9-8977-03035cf31f3d.png">

#### Sidebar

<img width="161" alt="screen shot 2019-02-12 at 11 52 25 am" src="https://user-images.githubusercontent.com/992008/52663956-b9f3c200-2ebc-11e9-8cf2-c3500ef69759.png">

#### Workaround

```
"[Yi Light]": {
  "editorSuggestWidget.selectedBackground": "#CFD4EF",
  "list.activeSelectionBackground": "#CFD4EF",
}
```

**Comment by chibicode** [comment]
Also, it seems that the error text in light theme is in white, making it impossible to read (text above the red squiggle):

<img width="384" alt="screen shot 2019-02-12 at 12 02 01 pm" src="https://user-images.githubusercontent.com/992008/52664521-14415280-2ebe-11e9-989e-2dd9eb81e09d.png">

**Comment by wangweixuan** [comment]
Thanks for the suggestions!

I believe the dark purple background is a mistake. It is expected to be a brighter blue, and the text should be white.
The error text is due to Microsoft/vscode#3429, and I will make a workaround for it.

**Comment by chibicode** [comment]
@wangweixuan wow, thank you for a quick fix! This is great.

**Comment by chibicode** [comment]
<img width="201" alt="screen shot 2019-02-14 at 3 06 43 pm" src="https://user-images.githubusercontent.com/992008/52823547-311a8900-306a-11e9-9eb1-19d6821e0951.png">

@wangweixuan I noticed the black text / purple background problem on the "install" buttons on extension sidebar. This is a tiny issue but just wanted to report here for completeness.

I think `prominentForeground` needs to be changed to white for the light theme:
https://github.com/wangweixuan/yithemes/blob/master/src/workbench.ts#L328

**Comment by wangweixuan** [comment]
Thanks for reporting! Will be fixed in 0.10.1.

**Comment by astrolemonade** [comment]
Thank you for creating this theme!
